exercise_003_parameter_untupling/README.md

```diff
@@ -2,3 +2,47 @@
 
 ## Background
 
+The need to write a pattern-matching decomposition when mapping over a sequence 
+of tuples has always annoyed some Scala developers. So many might appreciate this 
+new feature.
+
+Consider you have a list of pairs
+```scala
+val pairs = List(1, 2, 3).zipWithIndex
+// pairs: List[(Int, Int)] = List((1,0), (2,1), (3,2))
+```
+
+If you want to map `pairs` to a list of `Int`s so that each pair of numbers is 
+mapped to their sum. The best way to do this in _Scala 2.x_ was with a pattern 
+matching anonymous function.
+
+```scala
+pairs map {
+  case (x, y) => x + y
+}
+// res1: List[Int] = List(1, 3, 5)
+```
+Dotty now allows us to write
+
+```scala
+pairs map {
+  (x, y) => x + y
+}
+// val res0: List[Int] = List(1, 3, 5)
+```
+or 
+```scala
+pairs.map(_ + _)
+// val res1: List[Int] = List(1, 3, 5)
+```
+
+## Steps
+
+- We have similar usage in the following files
```

Comment:
I would mention that:
- This new way of encoding (using parameter untupling) is optional and in some cases a matter of personal taste.
- Perhaps formulate _We have similar usage in the following files_ as something along the lines of _we can apply the new way of encoding in several places in the source code. Have a look for them the following files:_